nautobot/core/settings.py

```diff
@@ -116,6 +116,8 @@
 # Test runner that is aware of our use of "integration" tags and only runs
 # integration tests if explicitly passed in with `nautobot-server test --tag integration`.
 TEST_RUNNER = "nautobot.core.tests.runner.NautobotTestRunner"
+NUM_SLOW_TESTS = None
+ALWAYS_GENERATE_SLOW_REPORT = False
```

Comment:
Do we need this setting at all? If so, can we add a `TEST_` prefix to it to make its scope clearer?